cobigen-cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -317,13 +323,40 @@ public void generate(File inputFile, File inputProject, List<? extends Generable
                 logger.info("Generating increments for input '" + inputFile.getName() + "', this can take a while...");
                 report = cg.generate(input, finalTos, Paths.get(outputRootPath.getAbsolutePath()), false, utilClasses);
             }
-            ValidationUtils.checkGenerationReport(report);
+            if (isJavaInput && ValidationUtils.checkGenerationReport(report)) {
+                Set<Path> format = report.getGeneratedFiles();
+                Iterator<Path> itr = format.iterator();
+                if (itr.hasNext()) {
+                    try {
+
+                        try {
+                            InputStream inputStream = Files.newInputStream(itr.next());
+                            String unformatterContent = IOUtils.toString(inputStream);
+                            new Formatter().formatSource(unformatterContent);
+                        } catch (IOException e) {
+                            logger.error("Unable to read input file");
+                        }
+                    } catch (FormatterException e) {
+                        logger.error("No elements found");
+                    }
+                }
+
```

Comment:
Please create a new method called `formatJavaSources(Set<Path> generatedFiles)` and move it to utilities class `ParsingUtils`